crates/goose-mcp/src/developer/mod.rs

```diff
@@ -811,65 +868,28 @@ mod tests {
 
     #[tokio::test]
     #[serial]
-    async fn test_text_editor_size_limits() {
```

Comment:
did these tests get moved somewhere? if not we should add them back in for non-windows builds

Reply:
sneaky goose